Automattic/jetpack
# Search package: Add search auto_config CLI

#### Changes proposed in this Pull Request:
The PR proposes a `auto_config` CLI to allow the functionality to be used thru WPCLI. The functionality for now is only intended for the search E2E tests.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Go to a site with Jetpack Search subscription
* Use a non block theme (Block themes are supported thru #22613 )
* Enable Instant Search
* Remove search widgets from sidebars
* Remove overlay sidebar wigets
* Run `jetpack wp jetpack-search auto_config`
* Ensure the removed widgets are added again.



![image](https://user-images.githubusercontent.com/1425433/152264248-a3da6bd2-bac1-4ece-a101-81c83d7fed3b.png)
